README.md

```diff
@@ -235,3 +240,5 @@ TAG=gcr.io/${PROJECT}/dicom-export-adapter
 gradle dockerBuildImage -Pdocker_tag=${TAG}
 docker push ${TAG}
 ```
+
+Check [wiki](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/wiki/Running-Docker-image-locally) for instructions on how to run Import Adapter Docker image locally.
```

Comment:
Minor edit: "Instructions on how to run the Import Adapter Docker image locally are available on the [wiki.](...)"

Reply:
Ok